changelog/4.feature.rst

```diff
@@ -0,0 +1,2 @@
+Creation of SunpyPlotter class which allows for plotting of a GenericMap through pyvista.
```

Comment:
Can we sphinx interlink the SunpyPlotter, GenericMap and pyvista? 

It is not setup in the docs/conf.py. 
You will want to change the line that is:

```
intersphinx_mapping = {'https://docs.python.org/': None}
```
to something like:
```
intersphinx_mapping = {
    "python": (
        "https://docs.python.org/3/",
        (None, "http://www.astropy.org/astropy-data/intersphinx/python3.inv"),
    ),
    "numpy": (
        "https://numpy.org/doc/stable/",
        (None, "http://www.astropy.org/astropy-data/intersphinx/numpy.inv"),
    ),
    "matplotlib": (
        "https://matplotlib.org/",
        (None, "http://www.astropy.org/astropy-data/intersphinx/matplotlib.inv"),
    ),
    "astropy": ("https://docs.astropy.org/en/stable/", None),
    "pyvista": ("https://docs.pyvista.org/", None),
}
```
Mind, I've not tested this works.

Reply:
I'll give this a shot